### PR TITLE
closed loop, check org api

### PR DESCRIPTION
**Pull request by pondrejk**
automation for https://bugzilla.redhat.com/show_bug.cgi?id=1828549
Checks for manifest cert exposed in organization api endpoint

```
pytest tests/foreman/api/test_organization.py -k test_negative_check_org_endpoint
                                                     
tests/foreman/api/test_organization.py .                                                                      [100%]

=================================== 1 passed, 42 deselected, 2 warnings in 7.12s ====================================

```

### CODE REVIEW

**Comment by swadeley** [comment]
Hello @pondrejk 

Does this test require a manifest to be installed?

I did not see one installed under `class TestOrganization:`

Thank you

**Comment by pondrejk** [comment]
@swadeley good point, checking

**Review by JacobCallahan — CHANGES_REQUESTED** [review]

**Comment by pondrejk** [comment]
Added an organization with manifest

File: tests/foreman/api/test_organization.py [hunk 1]
```diff
@@ -131,6 +132,27 @@ def test_negative_create_with_same_name(self):
...
+        :CaseImportance: Critical
+        """
+        org = (
+            entities.Organization()
+            .search(query={'search': f'name="{DEFAULT_ORG}"'})[0]
+            .read_json()
+        )
+        orgstring = json.dumps(org)
+        assert 'BEGIN CERTIFICATE' not in orgstring
+        assert 'BEGIN RSA PRIVATE KEY' not in orgstring
```
[the remark this answers is not in the document]
Comment (pondrejk) [reply tests/foreman/api/test_organization.py:154]: Yes, there was a particular key, but checking just a subsection wouldn't catch cert re-appearing elsewhere, that's why I check the whole output

**Review by swadeley — APPROVED** [review]
ACK

**Review by JacobCallahan — APPROVED** [review]

**Pull request merged**